presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -213,6 +213,19 @@ as part of the table name, separated by a colon. For example:
 .. _full text query: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax
 
 
+X-Pack Authentication
```

Comment:
We should this to "Password Authentication". `X-Pack` is a plugin framework for Elasticsearch. From the point of view of Presto, this is about using simple user/password scheme to authenticate with ES.